ports/geckolib/glue.rs

```diff
@@ -338,6 +374,54 @@ pub extern "C" fn Servo_StyleSet_Drop(data: RawServoStyleSetOwned) -> () {
     let _ = data.into_box::<PerDocumentStyleData>();
 }
 
+
+#[no_mangle]
+pub extern "C" fn Servo_ParseProperty(property_bytes: *const u8,
```

Comment:
It seems like the generated bindings were hand-written here, since the signature in ServoBindingList.h is correct. I'll fix this.